src/Common/Update.php

```diff
@@ -119,12 +119,13 @@ public function orWhere($cond)
      *
      * @param string $col The column name.
      *
-     * @return $this
+     * @param array $value
      *
+     * @return $this
      */
-    public function col($col)
+    public function col($col, ...$value)
     {
-        return call_user_func_array(array($this, 'addCol'), func_get_args());
+        return empty($value) ? $this->addCol($col) : $this->addCol($col, $value[0]);
```

Comment:
Actually you can just use `$this->addCol($col, ...$params);` . I have tried https://3v4l.org/sJ6S0 . I can merge some of the PR's . The interface changes probably need more review from @pmjones .